pymc/distributions/continuous.py

```diff
@@ -1464,7 +1463,7 @@ def dist(cls, lam, *args, **kwargs):
         lam = at.as_tensor_variable(floatX(lam))
 
         # Aesara exponential op is parametrized in terms of mu (1/lam)
-        return super().dist([at.inv(lam)], **kwargs)
+        return super().dist([at.reciprocal(lam)], **kwargs)
```

Comment:
Why this change?

Reply:
I had the impression that inv was deprecated in favor of reciprocal, did I misunderstand? see https://github.com/aesara-devs/aesara/pull/1054/commits/1a6fecf9b4cd47b59727e9f01d6981c17018b291

Reply:
Oh I missed that! You are absolutely correct.